### PR TITLE
Wallet integration feature

### PR DESCRIPTION
**Pull request by Koushith**
## This PR includes a simple wallet integration

- Redesigned the login flow.
- Integrated the Wallet functionality.

## Todos : 

- Prettify the user registration form.

https://www.loom.com/share/c5271a6b273446d1a18d636bf643daa0

### CODE REVIEW

File: src/components/Login/SignUp/archive/Form1.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
```
Comment (koshikraj) [inline src/components/Login/SignUp/archive/Form1.js]: Remove this file if not used.

Comment (Koushith) [reply src/components/Login/SignUp/archive/Form1.js]: yeah, removed unused files.

**Comment by Koushith** [comment]
## The latest commit adds the following features:

- [x] Toast Notifications

- [x] Master Contract Integration

https://www.loom.com/share/24a4bd4c3d76466aadc2d3be4098a3e6

**Comment by koshikraj** [comment]
A great PR  for our onboarding :+1: LGTM.

**Pull request merged**